# Get Orthographic Images

string[] camera_suffixes

---

bin_picking/OrthographicImage[] images